tests/test_remove_python2_unicode_compatible.py

```diff
@@ -0,0 +1,37 @@
+"""
+Tests for verifying the output of the script edx_repo_tools/codemods/django3/remove_python2_unicode_compatible.py
+"""
+import os
+from unittest import TestCase
+import shutil
+import uuid
+import subprocess
+
+
+class Python2UnicodeCompatibleTests(TestCase):
+    """
+    Tests for the Python2_unicode_compatible.py output
+    """
+    def setup_local_copy(self, file_name):
+        """
+        Setup local copy of the sample file for tests
+        """
+        current_directory = os.path.dirname(__file__)
+        temp_file = os.path.join(current_directory, 'sample_files', str(uuid.uuid4()) + ".py")
+        local_file = os.path.join(current_directory,  'sample_files', file_name)
+        shutil.copy2(local_file, temp_file)
+        return temp_file
+
+    def test_python2_unicode_compatible_removed(self):
+        """
+        Test replace script on a file to make sure it is renaming it properly and
+        also adding request parameter to updated function call
```

Comment:
we're not adding `request` parameter in updated function parameters, are we?

Reply:
Updated the description.